OMCompiler/SimulationRuntime/c/simulation/solver/gbode_ctrl.c

```diff
@@ -202,6 +206,8 @@ void getInitStepSize(DATA* data, threadData_t* threadData, DATA_GBODE* gbData)
   } else {
     h0 = 0.01 * d0/d1;
   }
+  if (gbData->initialFailures>0)
+    h0 /= pow(10,gbData->initialFailures);
```

Comment:
This could get very small very quickly. Is a minimum step size handled in `gbode_singlerate` or `gbode_birate`?

Reply:
The function is only called twice from the main solver routine, if an assert happens in a row.

Reply:
So `h0` is always larger or equal to 1e-9? Is this a sensible lower border and is it possible to change in the future?

```
d0 =1e-5, d1=1
h0 = 0.01* d0/d1
h0 /= 10^2
```

Reply:
How do you know `d1 <= 1`?

Reply:
The initialization method has been taken from the literature (Hairer et.al.). I am not happy with the details and the procedure should be analyzed in detail (maybe by a bachelor thesis). For now it works quite ok, but in case where an assert occurs by the functionODE, the control is lost in gbode. The solver module just calls gbode once again and one needs some logic to reduce the time step, so that the same assert happens again!

Reply:
*not happens again